src/dbt_client/dbtCloudIntegration.ts

```diff
@@ -832,6 +832,10 @@ export class DBTCloudProjectIntegration
     }
   }
 
+  findPackageVersion(packageName: string) {
+    return "";
```

Comment:
I didn't think of dbt cloud, if on cloud we can never get the version and that is not an exception then the return value should indeed be string | undefined. Core integration should still throw an error if it cannot detect the version as this indicates a problem.